### DISCUSSION TITLE
Roadmap for H1 2024

### DISCUSSION THREAD

**Discussion by aronmolnar** [Syslifters/sysreptor#184] [Announcements]
After we told you how we [prioritize SysReptor features in 2024](https://github.com/Syslifters/sysreptor/discussions/181), let's get concrete.

Here's our roadmap for the next months. The measures are prioritized by importance and efforts.

* [x] Count active SysReptor installations / users
* [x] Version history diff view + consolidated history view per project
* [x] reptor CLI tool: Nessus, Burp import functionalities
* [x] Define default notes structure in report design (https://github.com/Syslifters/sysreptor/issues/113)
* [x] Import and export notes structures
* [x] In-app notifications about available updates (https://github.com/Syslifters/sysreptor/issues/141)
* [x] Share PDF reports and files with customers with link to SysReptor (https://github.com/Syslifters/sysreptor/issues/122)
* [x] Status flag for designs (in-progress, ready-for-review, needs improvement, finished, deprecated)
* [x] Concurrent editing of findings and notes
* [x] Commenting on text (https://github.com/Syslifters/sysreptor/issues/124, https://github.com/Syslifters/sysreptor/issues/97)

## Further ideas

This is a feature collection we think of addressing. It's not clear that we will ever implement them.
We will prioritize after the current roadmap is done:

* E-Mail notifications (https://github.com/Syslifters/sysreptor/issues/96)
* User permissions per project (https://github.com/Syslifters/sysreptor/issues/120)
* Plugin system (ideas: Cyberchef, password manager integration, vulnerability scan imports, etc.)
* Improvements in license key management
* SysReptor image to Docker Hub
* Versioning of designs
* In-app notifications about missing or failed backups
* Basic CRM functionalities (offers, invoices, etc.)
* Read-only users

**Comment by danymat** [comment]
Hello, thank you for this H1 roadmap (and the already provided releases). Just a small request: could you precise (both here and in the releases description) what are the pro-only functionalities and the community ones ?

**Reply by aronmolnar** [reply]
Yes we'll do this.
You find a brief comparison at https://docs.sysreptor.com/features-and-pricing/

**Reply by aronmolnar** [reply]
Done in https://github.com/Syslifters/sysreptor/discussions/238
I hope this helps.

**Reply by danymat** [reply]
Thanks, it is perfect. (indeed, on release of 2024.28 I was not aware that collaborative was a pro feature. One more reason to push towards pro, you smart pentesters 🕺🏻 )

**Reply by aronmolnar** [reply]
We neither, to be honest.  
It is not always obvious how or if to restrict features and if it's technically feasible.

**Comment by danymat** [comment]
That's a good development !
